### ISSUE TITLE
Morphologic abnormality of the renal tubules

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4997]
Definition:

**Comment by nicolevasilevsky** [comment]
we can use this pattern:
https://github.com/obophenotype/upheno/blob/master/src/patterns/dosdp-dev/abnormalMorphologyOfAnatomicalEntity.yaml

UBERON_0009773 'renal tubule'

**Comment by nicolevasilevsky** [comment]
Suggested text def:
Any unspecified morphological anomaly of the renal tubules.

**Comment by pnrobinson** [comment]
This term already exists:
https://hpo.jax.org/app/browse/term/HP:0000091
I will add the label above as a synonym

**Comment by pnrobinson** [comment]
done
